app/views/content/steps-to-become-a-teacher/_step_2_understand_funding.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Undergraduate and postgraduate teacher training course fees are around £9,250 per year.</p>
+
+<p>You can apply for tuition fee and maintenance loans, even if you already have a student loan.</p>
+
+<p>If you're interested in teaching certain subjects, you might be able to get up to £29k tax-free, which doesn't need to be paid back, to support you while you are training.</p>
```

Comment:
```suggestion
<p>If you're interested in teaching certain subjects, you might be able to get up to £29k tax-free  to support you while you're training. This money does not have to be paid back.</p>
```
worth splitting this up into 2 sentences to make it a bit more concise and simple?